Lil-Peep/src/styles/modules/featured-categories.scss

```diff
@@ -120,15 +126,17 @@
 
 @media only #{ $tablet-view } {
   .categories-main {
+    background-color:  $color-bg;
+    border-bottom: 1px solid  $color-bd;
```

Comment:
Properties should be ordered background-color, border, border-bottom, border-color, border-radius, border-style, display, font-family, justify-content, margin-left, margin-right, margin-top